README.md

```diff
@@ -33,3 +33,9 @@ Development
 -----------
 
 * Build - For dependencies and build instructions, refer to [BUILD.md](docs/BUILD.md) .
+
+Limitations
+-----------
+
+* No DNS support.
+* Should use one network view at infoblox NIOS for one deployment.
```

Comment:
Doesn't have Infoblox DNS Support.
For one Kubernetes deployment only one Infoblox Network view can be used.